collaboration/1.slack.md

```diff
@@ -0,0 +1,101 @@
+# Slack
+
+[Slack](https://slack.com/) is a chat we use to discuss about anything.
```

Comment:
I agree with @gabro , you will see buildo.slack.com but I'm not so sure everybody took a good look at slack.com and what it actually is..